local-cluster/tests/local_cluster.rs

```diff
@@ -3740,6 +3743,18 @@ fn test_kill_partition_switch_threshold_progress() {
 #[serial]
 #[allow(unused_attributes)]
 fn test_duplicate_shreds_broadcast_leader() {
+    run_duplicate_shreds_broadcast_leader(true);
+}
+#[test]
+#[serial]
+#[ignore]
+#[allow(unused_attributes)]
+fn test_duplicate_shreds_broadcast_leader_ancestor_hashes() {
```

Comment:
i've split out this test case explicitly as it was getting lost in the random nature of the test.
after fixing the flakiness regarding the gossip message propagation, i realized the setup for this test is not equipped to successfully serve an ancestor hashes request.

I don't think this PR should be blocked by this test, I plan to fix the setup in an upcoming PR:  https://github.com/solana-labs/solana/issues/34857

Reply:
seems fair to me!